websites/T/TuneIn/presence.ts

```diff
@@ -14,78 +14,84 @@ presence.on("UpdateData", async () => {
-      } else {
-        title = document.querySelector("#playerTitle").textContent;
-        author = document.querySelector("#playerSubtitle").textContent;
-        const audioTime =
-            document.querySelector("#scrubberElapsed").textContent,
-          audioDuration =
-            document.querySelector("#scrubberDuration").textContent,
-          timestamp1 = presence.timestampFromFormat(audioTime),
-          timestamp2 = presence.timestampFromFormat(audioDuration),
-          timestamps = presence.getTimestamps(timestamp1, timestamp2),
-          paused = document.querySelector(
-            ".player-play-button__playerPlayButton___1Kc2Y[data-testid='player-status-paused']"
-          )
-            ? true
-            : false;
+          data.state = author;
+          if (author.length > 128)
+            data.state = `${author.substring(0, 125)}...`;
 
-        data.details = title;
-        if (title.length > 128) {
-          data.details = title.substring(0, 125) + "...";
-        }
+          data.smallImageKey = "live";
+          data.smallImageText = (await strings).live;
+        } else {
+          title = document.querySelector("#playerTitle").textContent;
+          author = document.querySelector("#playerSubtitle").textContent;
+          const audioTime =
+              document.querySelector("#scrubberElapsed").textContent,
+            audioDuration =
+              document.querySelector("#scrubberElapsed").innerHTML,
+            timestamp1 = presence.timestampFromFormat(audioTime),
+            timestamp2 = presence.timestampFromFormat(audioDuration),
+            timestamps = presence.getTimestamps(timestamp1, timestamp2),
+            paused = document.querySelector(
+              ".player-play-button__playerPlayButton___1Kc2Y[data-testid='player-status-stopped']"
+            )
+              ? true
+              : false;
 
-        data.state = author;
-        if (author.length > 128) {
-          data.state = author.substring(0, 125) + "...";
-        }
+          data.details = title;
+          if (title.length > 128)
+            data.details = `${title.substring(0, 125)}...`;
 
-        (data.smallImageKey = paused ? "pause" : "play"),
-          (data.smallImageText = paused
-            ? (await strings).pause
-            : (await strings).play),
-          (data.startTimestamp = timestamps[0]),
-          (data.endTimestamp = timestamps[1]);
+          data.state = author;
+          if (author.length > 128)
+            data.state = `${author.substring(0, 125)}...`;
 
-        if (paused) {
-          delete data.startTimestamp;
-          delete data.endTimestamp;
-        }
+          (data.smallImageKey = paused ? "pause" : "play"),
+            (data.smallImageText = paused
+              ? (await strings).pause
+              : (await strings).play),
+            (data.startTimestamp = timestamps[0]),
+            (data.endTimestamp = timestamps[1]);
```

Comment:
use ; not (),

Reply:
Thats a prettier thing?

Reply:
yea its because you use . not ;

Reply:
![image](https://user-images.githubusercontent.com/34792838/128734583-9b7c0b75-354e-435b-8085-7778fb680996.png)
those should be ;